app/src/main/java/org/kiwix/kiwixmobile/wifi_hotspot/WifiHotspotManager.java

```diff
@@ -0,0 +1,111 @@
+package org.kiwix.kiwixmobile.wifi_hotspot;
+
+import android.content.Context;
+import android.net.wifi.WifiConfiguration;
+import android.net.wifi.WifiManager;
+import android.os.Build;
+import android.os.Handler;
+import android.util.Log;
+import androidx.annotation.RequiresApi;
+import androidx.appcompat.app.AlertDialog;
+import org.kiwix.kiwixmobile.R;
+import org.kiwix.kiwixmobile.webserver.ServerStateListener;
+import org.kiwix.kiwixmobile.webserver.WebServerHelper;
+
+import static org.kiwix.kiwixmobile.utils.StyleUtils.dialogStyle;
+
+/**
+ * WifiHotstopManager class makes use of the Android's WifiManager and WifiConfiguration class
+ * to implement the wifi hotspot feature.
+ * Created by Adeel Zafar on 28/5/2019.
+ */
+
+public class WifiHotspotManager {
+  private WifiManager wifiManager;
+  Context context;
+  WifiManager.LocalOnlyHotspotReservation hotspotReservation;
+  boolean oreoenabled = false;
```

Comment:
booleans auto initialise to false, this variable name should be in camelHumps

Reply:
This was fixed in a later commit.